app/assets/stylesheets/main.scss

```diff
@@ -62,6 +63,40 @@ body {
   box-shadow: inset 0 -5px 0 #b4e7f8;
 }
 
+/* start of styles for cards on index.html.erb page */
+.ice-index {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  overflow: scroll;
```

Comment:
We should remove this because it looks poorly having the scroll bars, we can remedy overflow in alternate ways to keep things looking sharp.